lightly/active_learning/scorers/classification.py

```diff
@@ -0,0 +1,31 @@
+from typing import *
+
+import numpy as np
+
+from lightly.active_learning.scorers.scorer import Scorer
+
+
+def entropy(probs: np.ndarray, axis: int = 1):
+    zeros = np.zeros_like(probs)
+    log_probs = np.log2(probs, out=zeros, where=probs > 0)
+    entropies = -1 * np.sum(probs * log_probs, axis=axis)
+    return entropies
+
+
+class ScorerClassification(Scorer):
+    def __init__(self, model_output: np.ndarray):
```

Comment:
Please describe the expected shape from `model_output` in the docstring. Are we talking about something like this?
`NxC` meaning we have N samples and C classes. 
Do we care about the range of the values? Should they be in the range of [0,1] or can they be logits?

Reply:
In the long run we should probably support [0, 1] and logits but let's choose one to start with

Reply:
@philippmwirth I do not know how to calculate the prediction entropy and prediction margin with logits, do you know any literature on that?

Reply:
Check out the documentation on the cross entropy loss [here](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html). But again, it's ok to add this later as long as we specify the input format clearly.

Reply:
So the cross entropy loss out of logits is basically computed by first applying a softmax, as it computes the `exp(x_i)/sum(exp(x))`. The softmax gives the probability distribution as we want it. Thus our users would need to run the logits through a softmax first before setting the model_output in the scorer.

Reply:
Yes, let's do it this way for the moment. The advantage from calculating the entropy from logits is improved numerical stability